fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -236,7 +314,14 @@ private SeekableInputStream cachedStream() throws IOException {
             try {
                 CacheEntry entry = contentCache.get(location(), k -> newCacheEntry());
                 Preconditions.checkNotNull(entry, "CacheEntry should not be null when there is no RuntimeException occurs");
-                return ByteBufferInputStream.wrap(entry.buffers);
+                // someone have deleted local files, but still in cache
+                if (!entry.isExistOnDisk()) {
```

Comment:
When does this happen？ the cache file has  deleted Manually？

Reply:
someone delete  file from local disk,  but  the key in cache is valid, so  add check.